test/byContract/IexecPoco/IexecPoco1.test.ts

```diff
@@ -163,28 +151,23 @@ describe('IexecPoco1', () => {
             // Deposit required amounts.
             await iexecWrapper.depositInIexecAccount(requester, dealPrice);
             await iexecWrapper.depositInIexecAccount(scheduler, schedulerStake);
-            // Check balances and frozen before.
-            expect(await iexecPoco.balanceOf(requester.address)).to.equal(dealPrice);
-            expect(await iexecPoco.frozenOf(requester.address)).to.equal(0);
-            expect(await iexecPoco.balanceOf(scheduler.address)).to.equal(schedulerStake);
-            expect(await iexecPoco.frozenOf(scheduler.address)).to.equal(0);
             // Sign and match orders.
             const startTime = await setNextBlockTimestamp();
-            await signOrders(iexecWrapper.getDomain(), orders, ordersActors);
+            await signOrders(iexecWrapper.getDomain(), fullConfigOrders, ordersActors);
             const { appOrderHash, datasetOrderHash, workerpoolOrderHash, requestOrderHash } =
-                iexecWrapper.hashOrders(orders);
-            const dealId = getDealId(iexecWrapper.getDomain(), orders.requester, taskIndex);
+                iexecWrapper.hashOrders(fullConfigOrders);
+            const dealId = getDealId(iexecWrapper.getDomain(), fullConfigOrders.requester);
             expect(
                 await IexecPocoAccessors__factory.connect(proxyAddress, anyone).computeDealVolume(
-                    ...orders.toArray(),
+                    ...fullConfigOrders.toArray(),
                 ),
-            ).to.equal(volume);
+            ).to.equal(botVolume);
 
-            expect(await iexecPocoAsRequester.callStatic.matchOrders(...orders.toArray())).to.equal(
-                dealId,
-            );
-            const tx = iexecPocoAsRequester.matchOrders(...orders.toArray());
-            // Check balances and frozen after.
+            expect(
+                await iexecPocoAsRequester.callStatic.matchOrders(...fullConfigOrders.toArray()),
```

Comment:
Requester is not required in most tests:
```suggestion
                await iexecPoco.callStatic.matchOrders(...fullConfigOrders.toArray()),
```